src/kakarot/instructions/environmental_information.cairo

```diff
@@ -55,6 +56,39 @@ namespace EnvironmentalInformation {
         return ctx;
     }
 
+    // @notice ORIGIN operation.
+    // @dev Get execution origination address.
+    // @custom:since Frontier
+    // @custom:group Environmental Information
+    // @custom:gas 2
+    // @custom:stack_consumed_elements 0
+    // @custom:stack_produced_elements 1
+    // @return The pointer to the updated execution context.
+    func exec_origin{
+        syscall_ptr: felt*,
+        pedersen_ptr: HashBuiltin*,
+        range_check_ptr,
+        bitwise_ptr: BitwiseBuiltin*,
+    }(ctx: model.ExecutionContext*) -> model.ExecutionContext* {
+        
+        %{
+        import logging
+        logging.info("0x32 - ORIGIN")
+        %}
+
+        // Get Starknet address
+        /// TODO: get EVM address from Starknet address
+        let (tx_info) = get_tx_info();
+        let origin_address = Helpers.to_uint256(tx_info.account_contract_address);
```

Comment:
update with the accounts registry 
```suggestion
       tx_info.account_contract_address -> evm address
```

Reply:
thank you for the feedback. I have been trying to import the function "get_evm_address" from the account_registry.cairo file but i wasn't able to do so, which would be the right way to use it?.

Reply:
Check the balance opcode to see how to map the address

Reply:
Already fixed, thank you!